web/src/js/app.js

```diff
@@ -250,7 +256,7 @@ function initComposeModal() {
     $('#composer').modal('toggle');
   });
   const tokenfieldConfig = {
-    delimiter: [',', ';', ' '], // Must be in sync with SplitFunc (utils.go)
+    delimiters: [',', ';', ' '], // Must be in sync with SplitFunc (utils.go)
```

Comment:
Is the change from `delimiters` to `delimiter` accidental?

It [looks like](https://sliptree.github.io/bootstrap-tokenfield/) it should be `delimiter`. Please test and verify.

Reply:
I completely missed this and the above comment, I think some overzealous spam filter drops things from github somtimes.

The change is on purpose.  If you say "delimiter" it will only use one of them.  You have to use "delimiters" to get all of them to work.

I'll fix the non-idiomatic bool and resubmit.